### ISSUE TITLE
Can't achieve accuracy for PreActResnet18, does any guy know proper hyperparameters?

### ISSUE THREAD

**Issue by zdhNarsil** [kuangliu/pytorch-cifar#76]
I adjust learning rate according to the author, however, I can only get a test accuracy of 93%.  
I didn't normalize when preprocessing dataset, which may be the cause. But in my current case I do need to remove normalize from my preprocessing transformation.
So, can anyone tell me the detailed hyperparameters (lr, lr decay, weight decay, momentum, etc.) with a no-normalization CIFAR10 dataset and PreActResnet18?
